packages/eslint-plugin-reselect-utils/src/utils/ruleTester.ts

```diff
@@ -0,0 +1,27 @@
+import { join, resolve } from 'path';
+import { TSESLint } from '@typescript-eslint/experimental-utils';
+
+export const createRuleTester = () => {
+  const tester = new TSESLint.RuleTester({
+    parser: require.resolve('@typescript-eslint/parser'),
+    parserOptions: {
+      ecmaVersion: 2020,
+      project: join(__dirname, '../../tsconfig.eslint.json'),
+      sourceType: 'module',
+    },
+  });
+  const filename = resolve(__dirname, '../../file.ts');
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { run } = tester;
```

Comment:
you can try use something like this `const run = tester.run.bind(tester)`

Reply:
fixed